docs/running-on-kubernetes.md

```diff
@@ -120,8 +120,8 @@ This URI is the location of the example jar that is already in the Docker image.
 ## Client Mode
 
 Starting with Spark 2.4.0, it is possible to run Spark applications on Kubernetes in client mode. When running a Spark
-application in client mode, a separate pod is not deployed to run the driver. When running an application in
-client mode, it is recommended to account for the following factors:
+application in client mode, a separate pod is not deployed to run the driver. Your Spark driver does not need to run in
+a Kubernetes pod. When running an application in client mode, it is recommended to account for the following factors:
```

Comment:
How about changing `a separate pod is not deployed to run the driver. Your Spark driver does not need to run in
+a Kubernetes pod. When running an application in client mode` into `you may run your Spark driver outside the Kubernetes cluster or in a pod inside the cluster`?

Reply:
Think that's a bit wordy - perhaps, "When your application runs in client mode, the driver process is run locally. The driver can run inside a pod or on a physical host."

Reply:
I think you can even remove `the driver process is run locally`.